### DISCUSSION TITLE
[BUG] DNS_PROBE_FINISHED_NXDOMAIN when accessing to your website

### DISCUSSION THREAD

**Discussion by Zakichanu** [heyform/heyform#18] [Q&A]
![image](https://github.com/heyform/heyform/assets/49374657/8c96ebdf-acc7-48e7-a742-5c10f64f1780)

**Comment by dearroy** [comment]
Hello @Zakichanu,

Thank you for bringing this issue to our attention!

To help us further investigate, could you please provide the following information:
- Your ISP (Internet Service Provider)
- Your DNS provider
- Are you using a VPN to access our website?

**Comment by Zakichanu** [comment]
Hi @dearroy,

I Figured it out! I didn't thought it was my ISP which cas blocking your website... Sorry for this inconvenient.

**Comment by dearroy** [comment]
Hi @Zakichanu , It's strange that our website is being blocked by an ISP.

Could you please provide us with the name of the ISP so that we can contact them and request to have the ban lifted?

**Comment by Zakichanu** [comment]
Probably because it was on a company wifi, the IPS was `CELESTE`

**Comment by HelloGithubUniverse** [comment]
Your emails from the cloud Service are probably also blocked. I dont receive Emails on my icloud email (I tried account deletion , a code should be sent, and i tried data request). What is your Cloud Provider? Where is it located?

**Reply by dearroy** [reply]
We use Amazon SES for emails, but we have never experienced any blocking issues.

> I tried account deletion , a code should be sent, and i tried data request

Can you check if any other emails arrive?

**Reply by HelloGithubUniverse** [reply]
The email account itself is working. I used register/Login via apple and email is a Apple private relay forwarding.

**Reply by HelloGithubUniverse** [reply]
@dearroy are you abel to reproduce it?

**Reply by dearroy** [reply]
I haven't had a chance to check on that yet. We are currently taking a three-day break and will be back on Sunday to address it.

I will keep you updated!

**Reply by HelloGithubUniverse** [reply]
Any update yet?

**Comment by Zakichanu** [comment]
Hi @dearroy , I got the error again when sending a form... Tell me if there is anything to do please.
![image](https://github.com/heyform/heyform/assets/49374657/3adf3101-565c-436e-b922-3ed3bd8009f0)

**Comment by henriqueArrazao** [comment]
Same happened with me, using the WIFI didn't work, remote internet worked.
